views/search_view/test.py

```diff
@@ -35,30 +36,30 @@ def test_post_course_search_without_semester(self):
         self.assertContains(response, "CS101")
 
 
-class TestSearchUsers(TestCase):
+class TestSearchUsersAPI(TestCase):
```

Comment:
move these tests to the api file. And make sure to call the api, not the controller.